features/guidance/motorway.feature

```diff
@@ -176,11 +176,11 @@ Feature: Motorway Guidance
             | chi    | motorway_link |
 
        When I route I should get
-            | waypoints | route           | turns                                |
-            | a,e       | abcde,abcde     | depart,arrive                        |
-            | f,e       | fgc,abcde,abcde | depart,merge slight left,arrive      |
-            | a,i       | abcde,chi,chi   | depart,off ramp slight right,arrive |
-            | f,i       | fgc,chi,chi     | depart,off ramp right,arrive        |
+            | waypoints | route           | turns                              |
+            | a,e       | abcde,abcde     | depart,arrive                      |
+            | f,e       | fgc,abcde,abcde | depart,merge slight left,arrive    |
+            | a,i       | abcde,chi,chi   | depart,on ramp slight right,arrive |
```

Comment:
You are exiting the motorway abcde at this point. So chi is an off ramp (its not about getting on / off the ramp but whether you are leaving / entering a motorway)